index.js

```diff
@@ -1,51 +1,53 @@
-    const data = JSON.parse(localStorage.getItem('books')) || [];
-
-    for (let i = 0; i < data.length; i += 1) {
-      const bookContainer = document.createElement('div');
-      bookContainer.classList.add('book-wrap');
-      listBooks.appendChild(bookContainer);
-
-      const wrap = document.createElement('div');
-      wrap.classList.add('wrap');
-      bookContainer.appendChild(wrap);
-
-      const tittleBook = document.createElement('p');
-      tittleBook.classList.add('tittle-book');
-      tittleBook.innerText = `"${data[i].tittle}"`;
-      wrap.appendChild(tittleBook);
-
-      const authorBook = document.createElement('p');
-      authorBook.classList.add('author-book');
-      authorBook.innerText = `by ${data[i].author}`;
-      wrap.appendChild(authorBook);
-
-      const removeBtn = document.createElement('button');
-      removeBtn.classList.add('btn-remove');
-      removeBtn.innerText = 'Remove';
-      bookContainer.appendChild(removeBtn);
-
-      removeBtn.addEventListener('click', (parameter) => {
-        if (parameter.target.classList.value === 'btn-remove') {
-          listBooks.removeChild(parameter.target.parentElement);
-        }
-      });
-    }
-  });
-
-  // display sections
-  navList.addEventListener('click', displayList);
-  navAddNew.addEventListener('click', displayAddNew);
-  navContact.addEventListener('click', displayContact);
-};
-
+import { addBook } from './modules/createBooks.js';
+import { displayList, displayAddNew, displayContact } from './modules/functions.js';
+import {
+  form, navList, navAddNew, navContact, listBooks,
+} from './modules/selectors.js';
+
+const eventListeners = () => {
+  // add book
+  form.addEventListener('submit', addBook);
+
+  // book DOM
+  document.addEventListener('DOMContentLoaded', () => {
+    const data = JSON.parse(localStorage.getItem('books')) || [];
+
+    for (let i = 0; i < data.length; i += 1) {
+      const bookContainer = document.createElement('div');
+      bookContainer.classList.add('book-wrap');
+      listBooks.appendChild(bookContainer);
+
+      const wrap = document.createElement('div');
+      wrap.classList.add('wrap');
+      bookContainer.appendChild(wrap);
+
+      const tittleBook = document.createElement('p');
+      tittleBook.classList.add('tittle-book');
+      tittleBook.innerText = `"${data[i].tittle}"`;
+      wrap.appendChild(tittleBook);
+
+      const authorBook = document.createElement('p');
+      authorBook.classList.add('author-book');
+      authorBook.innerText = `by ${data[i].author}`;
+      wrap.appendChild(authorBook);
+
+      const removeBtn = document.createElement('button');
+      removeBtn.classList.add('btn-remove');
+      removeBtn.innerText = 'Remove';
+      bookContainer.appendChild(removeBtn);
+
+      removeBtn.addEventListener('click', (parameter) => {
+        if (parameter.target.classList.value === 'btn-remove') {
+          listBooks.removeChild(parameter.target.parentElement);
```

Comment:
- [ ] Here is a candidate for a reusable module as a similar code was repeated in createBook.js. I strongly suggest you create a module and export a function that will take an array and implement these methods, then import it for use in the two files.